src/Modules/OrchardCore.Commerce/Views/OrderPart.cshtml

```diff
@@ -61,23 +61,23 @@
         {
             <table class="table">
                 <tr>
-                    <th scope="col">@T["Kind:"]</th>
+                    <td><strong>@T["Kind:"]</strong></td>
```

Comment:
I'm not sure why you change `th` to `td` which is the preferred tag for table headers?!!

Reply:
I agree. If there is a reason why this is necessary, please include a comment.

Reply:
`td` by default makes the text bold

Reply:
I adjusted this because the previous version wasn't compliant with accessibility checks, making certain tests fail.
![image](https://github.com/OrchardCMS/OrchardCore.Commerce/assets/51411356/ee3b52f3-b5c3-4bfe-9f90-bf39b05b0258)
https://dequeuniversity.com/rules/axe/4.5/th-has-data-cells?application=axeAPI
The options were to either change the table's structure to include a header row or to somehow preserve the existing structure by making other changes. I went with the latter, though it's possible a better solution exists; I'm open to alternative suggestions.

> `td` by default makes the text bold

I'm not seeing so.
![image](https://github.com/OrchardCMS/OrchardCore.Commerce/assets/51411356/3bdbdf94-7e0c-4319-a2f1-c6e3f221f4b6)

Reply:
Sorry I mean `th`

Reply:
You get that validation error because these are row headers, but the `scope` attribute identifies them as column headers so the validator thinks you have several headers in one column. Please revert this change and replace all the `scope="col"` with `scope="row"`.